src/componentsLinks.ts

```diff
@@ -14,13 +14,16 @@ export const registerComponentsLinks = () => {
 
     vscode.languages.registerDocumentLinkProvider('vue', {
         async provideDocumentLinks(document) {
-            const activeEditor = vscode.window.activeTextEditor!
             if (!getExtensionSetting('enableLinks')) return
             // eslint-disable-next-line unicorn/no-await-expression-member
             const webpackAliases = getExtensionSetting('enableWebpackAliases') ? (await readWebpackAliases())?.aliases ?? [] : []
 
             const text = document.getText()
-            const matches: RegExpMatchArray[] = [...text.matchAll(/(import (.+) from )(['"].+['"])/g)]
+            const matches: RegExpMatchArray[] = [
+                ...text.matchAll(/(import (.+)\s+from )(['"].+['"])/g),
+                ...text.matchAll(/(\s*(?:let|const)\s*(\w+).*\((?:\/\*.*\*\/\s*)?)(['"].+['"])\)/g),
```

Comment:
Just curios why `.*` instead `import`? I believe it can lead to a lot of false positives? Also let's make `.*` within comment lazy  ( add ?)

Reply:
hope, I got you right